scripts/src/Platform/OSX.hs

```diff
@@ -1,14 +1,16 @@
 module Platform.OSX where
```

Comment:
This may be a touch nitpicky, but I recommend we call this platform `MacOS` to match with existing branding now.